src/routes/LandingPage.tsx

```diff
@@ -235,12 +239,7 @@ export default function LandingPage(): ReactElement {
     const publicationElement = project.publicationLink ? (
       <p>
         Related publication:{" "}
-        <a
-          href={project.publicationLink.toString()}
-          target="_blank"
-          rel="noopener noreferrer"
-          style={{ color: "var(--color-text-link)" }}
-        >
+        <a href={project.publicationLink.toString()} target="_blank" rel="noopener noreferrer">
```

Comment:
maybe over engineering: but I like your `HeaderLink` setup, seems like you could generalize it to have have the "openInNewTab" and then separate the styles

Reply:
Ah, that would be nice... I'll leave this as-is for now but next time I need to add more external links I'll think about making a component for it so there isn't so much repeated boilerplate. Thanks!